package.json

```diff
@@ -109,7 +109,7 @@
     "@babel/register": "^7.18.9",
     "@skyscanner/bpk-foundations-web": "^13.6.0",
     "@skyscanner/bpk-svgs": "^16.1.0",
-    "@skyscanner/stylelint-config-skyscanner": "^6.0.2",
+    "@skyscanner/stylelint-config-skyscanner": "^6.1.0",
```

Comment:
This version will now throw a warning that it is deprecated and recommend to bump to 7.0.0 (which includes the same changes just correctly semver)

https://github.com/Skyscanner/stylelint-config-skyscanner/blob/main/CHANGELOG.md#610-deprecated